assets/packages/limesurvey/survey.js

```diff
@@ -433,6 +433,9 @@ function activateSoftMandatory(){
 }
 
 function resetQuestionTimers(sid) {
+    if(!window.localStorage) {
```

Comment:
Then need to create a new mantis issue : Question timer didn't work without Local Storage :)

Reply:
I think we should just accept that timers will not work for browsers without localStorage? No localStorage seems to be a very low probability event, no? 

Bottom line here is I would rather have questionTimers malfunctioning quietly rather than whole javascript crashing on the page.

Reply:
> Bottom line here is I would rather have questionTimers malfunctioning quietly rather than whole javascript crashing on the page.

I accept the update for this reason :). This commit **must be merged** before creating another manitis issue :)

> I think we should just accept that timers will not work for browsers without localStorage? No localStorage seems to be a very low probability event, no?

Maybe, but then : must update manual and help of settings. Need a mantis issue for discussion.